tools/language-server/wstest/set-expression-value/README.md

```diff
@@ -0,0 +1,62 @@
+# set-expression-value
+
+Benchmark for command that sets the expression value.
+
+## Run
+
+Build Enso distribution.
+
+```bash
+sbt buildEngineDistribution
+```
+
+Start the Language Server and redirect ouput to `language-server.log`
+
+```bash
+built-distribution/enso-engine-0.0.0-dev-linux-amd64/enso-0.0.0-dev/bin/enso \
+  --log-level trace \
+  --server \
+  --root-id 6f7d58dd-8ee8-44cf-9ab7-9f0454033641 \
+  --path tools/language-server/wstest/set-expression-value/Unnamed/ \
+  --rpc-port 30616 \
+  --data-port 30717 \
+  &| tee language-server.log
```

Comment:
`&|` doesn't seem to work on my Ubuntu 22.04 `bash`. I had to use ` 2>&1| tee language-server.log`